assets/prototype/domain/eventEditor/EventEditor.js

```diff
@@ -1,11 +1,19 @@
 import useInitQueries from '../eventEditor/containers/queries/useInitQueries';
+import useFetchPrices from '../eventEditor/containers/queries/useFetchPrices';
+import useFetchPriceTypes from '../eventEditor/containers/queries/useFetchPriceTypes';
 import DatesList from './datesList/DatesList';
 import TicketsList from './ticketsList/TicketsList';
 
 const EventEditor = ({ eventId }) => {
 	const { datetimes, datetimeError, loadingDates, tickets, ticketError, loadingTickets } = useInitQueries({
 		eventId,
 	});
+
+	useFetchPriceTypes();
+
+	const ticketIn = tickets.map(({ id }) => id);
+	useFetchPrices({ ticketIn });
```

Comment:
why not add this to `useInitQueries` as well?
we have the `tickets` array there and this component is primarily concerned with display

Reply:
I initially did that but it was getting bigger and I thought to keep them separate. I think we should create separate hooks - `useFetchDatetimes`, `useFetchTickets`, `useFetchPriceTypes`, `useFetchPrices` etc. and then combine them inside `useInitQueries`. Every hook will handle its dependencies itself.